### PR TITLE
fix(curriculum): update descriptions and tests for business card lab

### LINKED ISSUE #56868: [ Full Stack Cert ] - update descriptions and tests for business card lab
There are a few updates we need to make to the following business card lab 

Here is the file that needs updating

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md

List of updates

1. Remove all jQuery from hints section
2. All mentions of `margin`, `padding`, `class` and `div` need to be inside backticks (``). This mainly applies to the user stories, but also double check the hints as well.

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56868

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md [hunk 1]
```diff
@@ -95,55 +95,55 @@ const a_hover = gs('a:hover')?.textDecoration;
 assert.equal(a_hover, 'underline');
 ```
 
-You should have one main `div` with the class `business-card`.
+You should have one main `div` with the `class` `business-card`.
 
 ```js
 assert(document.querySelector('div')?.getAttribute('class') === 'business-card');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md:101]:
```suggestion
assert.strictEqual(document.querySelector('div')?.getAttribute('class'), 'business-card');
```

File: curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md [hunk 2]
```diff
@@ -95,55 +95,55 @@ const a_hover = gs('a:hover')?.textDecoration;
...
 
 ```js
 assert.equal(new __helpers.CSSHelp(document).getStyle('.business-card').getPropVal('width'), '300px');
 ```
 
-Your `.business-card` class should have a background color of your choice.
+Your `.business-card` `class` should have a background color of your choice.
 
 ```js
 assert(new __helpers.CSSHelp(document).getStyle('.business-card')?.backgroundColor);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md:113]:
```suggestion
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.business-card')?.backgroundColor);
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
I can't leave a comment on line 67 and 81, but those can become `assert.isTrue()`

**Comment by gagan-bhullar-tech** [comment]
@ilenia-magoni updated PR.

**Review by majestic-owl448 — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**